recipes-bsp/flashrom_git.bb

```diff
@@ -0,0 +1,19 @@
+DESCRIPTION = "flashrom is a utility for identifying, reading, writing, verifying and erasing flash chips"
+LICENSE = "GPLv2"
+HOMEPAGE = "https://github.com/pcengines/flashrom"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+DEPENDS = "pciutils libusb libusb-compat"
+
+SRC_URI = "git://github.com/pcengines/flashrom.git \
+           "
+SRC_URI[md5sum] = "91bab6c072e38a493bb4eb673e4fe0d6"
```

Comment:
@Asiderr This recipe will not build. When using the `git` fetcher, the SRC_URI[md5sum|sha256sum] are not used. They are used in case of downloaded files / tarballs. In case of git repo, SRCREV needs to be set: https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#var-SRCREV

Reply:
You should use https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#var-bb-PREFERRED_PROVIDER to tell which flashrom should be used.